<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MW</class>
 <widget class="QMainWindow" name="MW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Money List</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendar">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>296</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="moneyList">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAddItem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLog">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Caculator</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>210</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbDate">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbItem">
         <property name="text">
          <string>Item</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbMoney">
         <property name="text">
          <string>Money</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QDateEdit" name="date"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnToday">
           <property name="text">
            <string>Go Today</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="leItem"/>
       </item>
       <item>
        <widget class="QSpinBox" name="sbMoney">
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="singleStep">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnSort">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>310</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort By Date</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionNew_database"/>
    <addaction name="actionConnect_database"/>
    <addaction name="separator"/>
    <addaction name="actionConnect_relative_database"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOption"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect_database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/img/open.ico</normaloff>:/img/img/open.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/img/new.ico</normaloff>:/img/img/new.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;New database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/img/exit.ico</normaloff>:/img/img/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnect_relative_database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/img/connect.ico</normaloff>:/img/img/connect.ico</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect relative database</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leItem</tabstop>
  <tabstop>sbMoney</tabstop>
  <tabstop>btnAddItem</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>date</tabstop>
  <tabstop>btnToday</tabstop>
  <tabstop>calendar</tabstop>
  <tabstop>moneyList</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
